testing/test_metrics.py

```diff
@@ -0,0 +1,67 @@
+import ivadomed.metrics as imed_metrics
+import numpy as np
+
+
+def test_multi_class_dice_score():
```

Comment:
That's great!
At some point we may want to move to "pytest.mark.parametrize", similar to what is done in `test_losses.py`... opening an issue.